src/routes/github/index.js

```diff
@@ -22,7 +22,7 @@ const routes = (config) => [
 			const {
 				repository: { full_name: repositoryFullName },
 			} = payload;
-			const streamlabs = new StreamLabs({ token: config.STREAMLABS_TOKEN });
+			const streamlabs = new StreamLabs({ token: config.STREAMLABS_TOKEN }, request);
```

Comment:
Can we only pass the log method?

Reply:
Not without binding it to the original object

```javascript
{log: request.log.bind(request)}
```

Are you okay with this?

Reply:
Will revise when we implement #50